RoboJackets/igvc-software
# Feat/scan to pointcloud header cpp

# Description

Refactors the scan_to_pointcloud node into a header file and C++ class.

Fixes #{{Was not properly formatted in earlier submission}}

# Testing steps (If relevant)
1. Run simulation on master branch
2. Run simulation on feat/scan_to_pointcloud_header_cpp branch
3. Check that the results for the scan_to_pointcloud node are the same for both branches.

Expectation: The scan_to_pointcloud node should maintain functionality

# Self Checklist
